### ISSUE TITLE
Reduce default thumbnail size

### ISSUE THREAD

**Issue by Beep6581** [Beep6581/RawTherapee#3535]
I suggest quickly changing the default thumbnail size for the RT5 release from `ThumbnailSize=240` to `ThumbnailSize=150`.

This is what users will see when they install RT5 and open a photo on a common "full HD" (1920x1080) screen:
https://i.imgur.com/UPvUBff.jpg
It would be even worse on lower resolutions. They will not immediately know that they can press "t" to show the filmstrip toolbar from which they can make those thumbs smaller.

At 150 (two clicks smaller) things are far better balanced:
https://i.imgur.com/1jjKNVT.jpg

**Comment by Beep6581** [comment]
On a related note, at some point in the past I opened a feature request that the thumbnail size of the thumbs in the Filmstrip be decoupled from the size of those in the File Browser, the idea being that you can have large thumbs in the File Browser but usually want small thumbs in the Filmstrip. There is such a feature, but it is disabled by default (Preferences > File Browser > Thumbnail Options > Same thumbnail height between the Editor's panel and the File Browser). Is this disabled by default for performance reasons? If the size of thumbs in the Filmstrip don't match those in the File Browser, are they cached twice, or are the larger ones cached and the smaller ones downscaled on the fly from the larger ones?

The current defaults are:
```
ThumbnailSize=240
ThumbnailSizeTab=180
ThumbnailSizeQueue=160
SameThumbSize=1
```

If we were to disable sameThumbSize by default, I would suggest these defaults:
```
ThumbnailSize=150
ThumbnailSizeTab=150
ThumbnailSizeQueue=150
SameThumbSize=0
```

If we were to not disable SameThumbSize, I would still suggest changing the defaults because it doesn't make sense that the sizes between *Tab and *Queue should differ by 20px:
```
ThumbnailSize=150
ThumbnailSizeTab=150
ThumbnailSizeQueue=150
SameThumbSize=1
```

Finally, I would replace the SameThumbSize label
```
Same thumbnail height between the Editor's panel and the File Browser
```
with
```
Same thumbnail height between the Filmstrip and the File Browser
```

**Comment by Floessie** [comment]
@Beep6581 Maybe I don't get it, but what's the difference to `SameThumbSize=1` here?

```
ThumbnailSize=150
ThumbnailSizeTab=150
ThumbnailSizeQueue=150
SameThumbSize=0
```

**Comment by Beep6581** [comment]
@Floessie it's only a usability thing. New users won't know about that option in Preferences. If I were new to RT I would not assume that the size of thumbs in the File Browser is locked to be the same as those in the Filmstrip because the two play a different role. Personally I would find 150 a good size for both the Filmstrip and the File Browser, but I know from the forum that some people like large File Browser thumbs, so they will click "+" and may be surprised that now their Filmstrip covers half the screen, and then they will come running to the forum asking how to change that... So the only argument I have for `SameThumbSize=0` is to prevent that.

**Comment by Floessie** [comment]
@Beep6581 Okay, now I see. Thanks.

**Comment by Beep6581** [comment]
@Floessie but do you [agree]/[disagree]/[don't care]? I need to know to make a decision.

**Comment by Beep6581** [comment]
I went on a limb and assumed 160 might be marginally faster as it's divisible by 8.

Pull request:
https://github.com/Beep6581/RawTherapee/pull/3536/files

**Comment by Floessie** [comment]
@Beep6581 I agree that your proposal makes sense. But I don't care, as I'm used to select the image in the file browser and work on it with the thumbs panel above the editor panel hidden. For me the file browser thumbs are all about indentification, so their sizes don't matter to me as long as I can discern them. :smile:

**Comment by heckflosse** [comment]
:+1: for the changes.

**Comment by Hombre57** [comment]
`SameThumbSize=1` is an optimization feature as well, since RT won't have to reprocess the thumbnails when switching back and forth between the Editor panel and the File panel.

**Comment by Beep6581** [comment]
I mentioned that, but do you have any numbers? I don't feel any difference in speed and I'm not on a fast machine.

**Comment by Hombre57** [comment]
I don't have numbers, but it obviously depend on the number of image in your directory. With 200 file, you should notice the difference I guess.

**Comment by Hombre57** [comment]
Btw, performance and cache size on disk for thumbnails depend on `MaxPreviewHeight` as well. The smaller the faster, but of course the user will have to change this value in Preference if he want to get bigger thumbnail size. Mine is set to 200px.

**Comment by heckflosse** [comment]
Mine is set to 120px which is already larger than my thumbnail ;)

**Comment by Beep6581** [comment]
@Hombre57 we already decreased `MaxPreviewHeight` (`maxThumbnailHeight`) from 400 to 250 in May 2015, in commit b192a7f012a036f3c84e8c59450272c872bd57a1.

I set all sizes to 160 as in the PR.
With SameThumbSize=1 it took 66 seconds to generate 200 thumbs in the File Browser (3 runs, cache deleted every time). Clicking on the Editor was instant. Back in the File Browser, clicking "+" twice to enlarge them to 250 was also instant. Clicking back to the Editor was instant.
With SameThumbSize=0 it took the same 66 seconds to generate 200 thumbs in the File Browser (3 runs, cache deleted every time). Clicking on the Editor was instant. Back in the File Browser, clicking "+" twice to enlarge them to 250 was also instant. Clicking back to the Editor was instant.

There must be a speed difference, but not one I could observe.

I would not like to spend more time on this. There were no objections to this part of the PR:
```
ThumbnailSize=160
ThumbnailSizeTab=160
ThumbnailSizeQueue=160
```

I believe setting `SameThumbSize=0` may improve the user experience for reasons mentioned in [this comment](https://github.com/Beep6581/RawTherapee/issues/3535#issuecomment-266981087) and that is the only reason I suggest doing that. Shall we change it or leave it?
